tests/testthat/test-use-standalone.R

```diff
@@ -0,0 +1,39 @@
+test_that("can import standalone file with dependencies", {
```

Comment:
`skip_if_offline()`?

Reply:
I used `skip_on_cran()` to be consistent with the other test. But should I switch both to use `skip_if_offline()`?

Reply:
I suspect the other test probably first appeared in the era before `skip_if_offline()` implied `skip_on_cran()`. So yeah I think `skip_if_offline()` is better for both now.

Reply:
I just did this.